### ISSUE TITLE
[Fixed in GH-6052] Missing Visual Focus on Section1 Secondary Banner

### ISSUE THREAD

**Issue by developpeurweb** [SAP/cloud-commerce-spartacus-storefront#4294]
![Screen Shot 2019-08-28 at 10 11 38 PM](https://user-images.githubusercontent.com/15113219/63905048-f910dd00-c9e0-11e9-80ab-fc6d736d8107.png)
Focus it's either missing or hidden behind the image as it seems to be the case for other banners

**Comment by developpeurweb** [comment]
**For QA:**

- To test in Safari please refer to [these instructions](https://github.com/SAP/cloud-commerce-spartacus-storefront/issues/5041#issuecomment-558883497).
- This got fixed in ticket GH-6052
- Theme `calydon` should be enabled to see changes.
- Visible focus outline overlapping adjacent ui elements may happen and may not be an issue, if in doubt please open a ticket per element or reach out to UI developer.
- Please separate visible focus issues (CSS: element without outline) from tabbing issues (HTML/TS: element not reachable with keyboard).

![Screen Shot 2020-01-31 at 4 40 47 PM](https://user-images.githubusercontent.com/15113219/73576683-0b3bef00-4449-11ea-9f83-fd84835f7625.png)

**Comment by KateChuen** [comment]
Tested on dev3.
- Chrome ✅
- Firefox ✅
- Safari ✅
